### PR TITLE
Added genres to fixture

### LINKED ISSUE #77: Update homepage to show visually if a venue has a review
## Overview
Find a way to show on the homepage whether or not a venue has a review.

### LINKED ISSUE #78: Allow users to add genres OR add more genres to the fixtures file
## Overview
Right now There are only three genres in the fixtures file And no way for the user to add more genres. We can fix this by adding many many more genres to the fixtures file. Another way to fix this is to allow users to add genres through a form. One is easier than the other but (in case someone is feeling ambitious) they are not mutually exclusive: we *could* do both.

### PR DESCRIPTION
**Pull request by jonfaustothen**
Added more genres to fixture file, also changed background color with this push to slate blue

### CODE REVIEW

**Review by Jonathanlouiskirk — DISMISSED** [review]

**Review by storrer — DISMISSED** [review]

**Review by Jonathanlouiskirk — DISMISSED** [review]

**Review by Jonathanlouiskirk — DISMISSED** [review]

**Review by storrer — CHANGES_REQUESTED** [review]
Looks good, but the review_form.html navbar logout/login `<li>` has been changed back to `review_create` which no longer functions the way it used to.

File: revenue/revenueapp/templates/revenueapp/review_form.html [hunk 1]
```diff
@@ -14,34 +14,35 @@
...
   </div>
 
-<body id="background">
-    
-<div class="navbar">
-   <ul>
-      <li><a href="{% url 'home' %}">Home</a></li>
-      <li><a href="{% url 'about' %}">About</a></li>
-      <li><a href="{% url 'venue_create' %}">Create A Venue</a></li>
-      <li><a href="{% url 'logout' %}?next={% url 'individual_venue' venue.id %}">Logout</a></li>
```
Comment (storrer) [inline revenue/revenueapp/templates/revenueapp/review_form.html:24]: Can you add this line back into the navbar for this page?

File: revenue/revenueapp/templates/revenueapp/review_form.html [hunk 2]
```diff
@@ -14,34 +14,35 @@
...
-</form>
-</section>
-</main>
+<body id="background">  
+   <div class="navbar">
+      <ul>
+         <li><a href="{% url 'home' %}">Home</a></li>
+         <li><a href="{% url 'about' %}">About</a></li>
+         <li><a href="{% url 'venue_create' %}">Create A Venue</a></li>
+         <li><a href="{% url 'review_create' %}">Review</a></li>
```
Comment (storrer) [inline revenue/revenueapp/templates/revenueapp/review_form.html]: This link no longer works and should be replaced.

File: revenue/revenueapp/templates/revenueapp/review_form.html [hunk 3]
```diff
@@ -14,34 +14,35 @@
...
-<img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR6sdT7cf98F4_QUJiEqs14XwqNaE-vZPzqRQ&usqp=CAU" alt="10 best venues in houston.com" width="60%" class="center">
-
-<h1 id="create_review_heading_text">Please leave a review</h1>
-
-<h1> {{ venue.name }} </h1>
-<form method="post">
-    {% csrf_token %}
-    {{ form.as_p }}
-    <input type="submit" value="Save">
-    <a href="{% url 'individual_venue' venue.id %}">Cancel</a>
```
Comment (storrer) [inline revenue/revenueapp/templates/revenueapp/review_form.html:41]: I also added a cancel button and the review's  name to the form, and it has been overwritten with an older version of this form.

**Review by storrer — APPROVED** [review]

**Review by storrer — CHANGES_REQUESTED** [review]
There are no merge conflicts to resolve so I'm un-approving this. You replaced all the changes I made to review_form.html...

**Review by Jonathanlouiskirk — APPROVED** [review]

**Review by storrer — APPROVED** [review]

**Pull request merged**